sfcasts/ep1/github-service-part-2.md

```diff
@@ -1,103 +1,81 @@
-uh, health status label. We're going to come up here to the top and we're going to
-set health = health status healthy. This is because anytime that a dinosaur does not
-have a sick label, we have to assume that the dinosaur is healthy. Uh, move back into
-her terminal and let's test this out vendor bend PHP unit. Awesome. We have eight
-tests and 11 assertions. Our service is working beautifully. We're going to do one
-last thing here. And that is at the top of our GitHub service. We want to log all of
-these HTTP requests we make to GitHub. This will come handy in production. So let's
-create a public function construct, and we're going to create a private logger
-interface and we'll name it logger. And then, uh, right before a four each loop after
-we call the respo or after we call the, uh, GitHub API, let's go ahead and
-
-Let's go ahead and do this logger. We want to log as info and our message will be
-request Dino issues. And we're going to add a little extra cont context to this
-message. And by passing an array, which will contain the Dino's name. So Dino, uh,
-for the key and the name is dinosaur name at what we always know, which, uh, Dino we
-were trying to get, uh, an issue from. And then below that, let's go ahead and throw
-in the response status. Ah, we're going to throw in the response status that was
-given back from our, uh, HTTP request. So response get status code, and we're going
-to return, uh, move back to her terminal. Let's run this test one more time. Oh no,
-we got two errors now. So both of our GitHub service test test get health report
-returns, correct health status for Dino with data sets, sick Dino and a healthy Dino
-are both saying that we have an argument count error, two few arguments to function
-at GitHub service construct, zero pass. And exactly one were expected looking at our,
-uh, service here. That's because we're supposed to be passing in from our test a
-logger instance, and we are not doing that yet. So coming up next, we'll show you how
-to do that. And we'll fix this.
+# GitHub Service - Part 2
 
+Now that we have an idea of what we need the `GithubService` to do, let's add
+the logic inside that we'll use to fetch the issues from the `dino-park` repository
+using GitHub's API.
+
+## Add the client and make a request
+
+In the terminal, add Symfony's HTTP Client to our app with:
+
+```terminal
+composer require symfony/http-client
+```
+
+Which we'll use to call the API. Inside of `GithubService`, instantiate an HTTP 
+client with `$client = HttpClient::create()`. To make a request, the
+`$client->request()` method. We need to tell the client 2 things, 1st: what HTTP
+method to use, like `GET` or `POST`. In this case, the `method` will be `GET`. 
+2nd: the `url` that we'll call is 
+`https://api.github.com/repos/SymfonyCasts/dino-park/issues` will get all the 
+"issues" from the `dino-park` repository that our service needs.
```

Comment:
```suggestion
-"issues" from the `dino-park` repository that our service needs.
+"issues" from the `dino-park` repository.
```

This takes practice: but always look to see what extra words are being added that don't contain value.